pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts

```diff
@@ -0,0 +1,370 @@
+import {
```

Comment:
This is not a test file so I would suggest it should go under test folder .
According to this cypress framework guidelines .. It should go under integration/types/tier1/config_tier1.ts

Reply:
Cypress framework guidelines -  https://docs.google.com/document/d/1IfApGDf1JmFHCf48XH3xk2noq6XJVBHv_tRuqMTqDBs/edit

Reply:
Thanks @sshveta for the review! This is the config file for the tier1 test cases which contains the different provider details, mappings to be used for the tests and all the plans essentials requirements for running tests.
In types we have exported the data types required for the configs.
And we have always declared the required config files in the Test folder of particular providers and tiers.

Reply:
@kpunwatk Okay if that's the framework guidelines your team follows . Please check with @ibragins too .
In MTA we keep only test files under test folder .